### PR TITLE
Travel api

### PR DESCRIPTION
**Pull request by MisaOgura**

### CODE REVIEW

File: lib/travel_api.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+require 'dotenv'
+require 'json'
+Dotenv.load
+
+class TravelApiCall
+
+  include HTTParty
+
+  def initialize
+    @base_uri = 'https://api.tfl.gov.uk/journey/journeyresults/1000004/to/1000241'
```
Comment (rhiannonruth) [inline lib/travel_api.rb]: this needs to take arguments rather than hardcoded to and from

File: severity codes [hunk 2]
```diff
@@ -0,0 +1,22 @@
+these are the codes used by tfl to represent the
```
Comment (rhiannonruth) [inline severity codes]: should this file be gitignored?

**Comment by rhiannonruth** [comment]
🚢

File: lib/travel_api.rb [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+require 'dotenv'
+require 'json'
+Dotenv.load
+
+class TravelApiCall
+
+  include HTTParty
+
+  def initialize(from, to)
+    @base_uri = "https://api.tfl.gov.uk/journey/journeyresults/#{from.to_s}/to/#{to.to_s}"
```
Comment (shaneoston72) [inline lib/travel_api.rb]: make the url an env variable.

**Comment by Managram** [comment]
:shipit:

**Pull request merged**